Framework/API/test/ExperimentInfoTest.h

```diff
@@ -467,7 +467,7 @@ class ExperimentInfoTest : public CxxTest::TestSuite {
     mappings.emplace(1, std::set<Mantid::detid_t>{2});
     expt.cacheDetectorGroupings(mappings);
 
-    size_t index;
+    size_t index = 0;
```

Comment:
While this is correct and I *don't* see any need to change it, part of me wants to use uniform initialization here. The newer syntax is a bit more restrictive about potential narrowing casts.